helion/runtime/settings.py

```diff
@@ -399,8 +399,12 @@ class _Settings:
     )
     autotuner_fn: AutotunerFunction = default_autotuner_fn
     autotune_baseline_fn: Callable[..., object] | None = None
-    autotune_baseline_atol: float = 1e-2
-    autotune_baseline_rtol: float = 1e-2
+    # Default to None, will be set to 1e-2 in __init__ if not user-specified
+    autotune_baseline_atol: float = dataclasses.field(default=1e-2)
+    autotune_baseline_rtol: float = dataclasses.field(default=1e-2)
+    # Internal fields to track if user explicitly set tolerance values
+    _user_set_atol: bool = dataclasses.field(default=False, init=False, repr=False)
+    _user_set_rtol: bool = dataclasses.field(default=False, init=False, repr=False)
```

Comment:
Not sure if it's possible, but would really love to avoid needing to have these private fields on the  settings class, to keep things more simple